cypress/integration/vote_for_card_spec.js

```diff
@@ -1,11 +1,14 @@
 describe('Vote for a card', () => {
     beforeEach(() => {
-        cy.login('unicorn');
-        cy.request(`http://localhost:12346/connect?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
-        .then(() => {
-            cy.startGame();
-            cy.request(`http://localhost:12346/playCardWord?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
-            .then(() => cy.playCard());
+        cy.login('unicorn').then(() => {
+            cy.request(`http://localhost:12346/connect?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
```

Comment:
You could return some of these promises if you wanted to chain them in a nicer way.